docusaurus/docs/cloud/advanced/email.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Email Provider
+displayed_sidebar: cloudSidebar
+description: Configure Strapi Cloud to use a third-party email provider.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/email.html
+---
+
+# Email Provider
+
+Strapi Cloud comes with a basic email provider out of the box. However, you can also configure it to utilize a third-party email provider, if needed.
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party email provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party email provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  Required Email Provider Plugin Configuration
+  // ...
+});
+```
+Each provider will have different configuration settings available. Review the respective entry for that provider in the [Marketplace](https://market.strapi.io/).
+
+Below are example configurations for the Email plugins.
+
+<Tabs groupId="email-examples" >
+
+<TabItem value="sendgrid" label="Sendgrid">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'sendgrid',
+      providerOptions: {
+        apiKey: env('SENDGRID_API_KEY'),
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem >
+<TabItem value="amazon-ses" label="Amazon SES">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'amazon-ses',
+      providerOptions: {
+        key: env('AWS_SES_KEY'),
+        secret: env('AWS_SES_SECRET'),
+        amazon: 'https://email.us-east-1.amazonaws.com',
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem>
+<TabItem value="mailgun" label="Mailgun">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'mailgun',
+      providerOptions: {
+        key: env('MAILGUN_API_KEY'), // Required
+        domain: env('MAILGUN_DOMAIN'), // Required
+        url: env('MAILGUN_URL', 'https://api.mailgun.net'), //Optional. If domain region is Europe use 'https://api.eu.mailgun.net'
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem>
+</Tabs>
+
+:::caution
+The email provider must be installed as a package dependency in your Strapi project. For example, to use the Sendgrid email provider, install the `@strapi/provider-email-sendgrid` package.
+:::
```

Comment:
It could be a bit confusing that this step should be done at the beginning of the configuration but is only mentioned at the end of the section (if I understood well) 🤔 

Maybe we could reorganise a bit so that the configuration steps are in the right order: 
1. Install package dependency
2. Create plugins.js or plugins.ts file with the right content
3. Configure available settings (refer to Marketplace)
What do you think? 😊